### PR TITLE
Improve serving, update missing output content

### PR DESCRIPTION
**Pull request by ReDeiPirati**

### CODE REVIEW

File: docs/examples/deep_corrector.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ This model should perform better on the given inputs compared to the previous on
 If you run a job with `--mode serve` flag, FloydHub will run the `app.py` file in your project and attach it to a dynamic service endpoint:
 
 ```bash
-floyd run --mode serve --env tensorflow-1.5 --data floydhub/deep-text-corrector/23/:input
+floyd run --mode serve --env tensorflow-1.5 --data <REPLACE_WITH_JOB_OUTPUT_NAME>:input
```
Comment (whatrocks) [inline docs/examples/deep_corrector.md]: what is `job_output_name` in the new, non-output workflow? Does this still make sense. Wouldn't it just be the job name?

File: docs/guides/serving.md [hunk 2]
```diff
@@ -90,14 +83,50 @@ URL to service endpoint: https://www.floydlabs.com/expose/mkxjJa46aJBdwP4AEdKxfU
...
+curl -o taipei_output.jpg -F "file=@./images/taipei101.jpg" https://www.floydlabs.com/serve/narenst/projects/fast-style-transfer
 ```
 
 ![Muse](../img/taipei_muse.jpg)
 
 You will see the default style ([la_muse](https://github.com/floydhub/fast-style-transfer/blob/master/examples/style/la_muse.jpg)) is applied to the input image.
 
+### Build an interactive website upon Serving
+
+Serving is not limited to single REST endpoint. You can build complex interactive website at its top. Here's an example.
```
Comment (whatrocks) [inline docs/guides/serving.md:106]: I think more details around how to do this, like we support basic Flask templating. Also note if there are any limits. And possibility, would be good to show some snippets for common UI components. This last bit might be out of scope for these changes, but I think they would help/

File: docs/examples/deep_corrector.md (hunk 1 above)
Comment (ReDeiPirati) [reply docs/examples/deep_corrector.md]: Good call! I'll fix it

**Review by houqp — APPROVED** [review]

**Comment by houqp** [comment]
I just noticed my fix for the serving doc is not pushed to master. I will wait for your PR to be merged, then add in my changes.

**Pull request merged**